scripts/dist/import.json

```diff
@@ -0,0 +1 @@
danger-fg);\n    color: var(--color-header-bg);\n    content: \"Unsupported GitHub theme detected! Please switch to the default light/dark theme via the GitHub Appearance settings to get the best experience for the Catppuccin GitHub userstyle.\";\n    z-index: 9999;\n  }\n\n  #coloredButton(@item, @color, @flat) {\n    & when (@flat) {\n      --color-btn-@{item}-text: @color;\n      --color-btn-@{item}-bg: @base;\n      --color-btn-@{item}-icon: @color;\n\n      --color-btn-@{item}-hover-text: @base;\n      --color-btn-@{item}-hover-bg: @color;\n      --color-btn-@{item}-hover-border: @color;\n\n      --color-btn-@{item}-selected-text: @base;\n      --color-btn-@{item}-selected-bg: @color;\n      --color-btn-@{item}-selected-border: @color;\n\n      --color-btn-@{item}-disabled-text: fadeout(@color, 60%);\n      --color-btn-@{item}-disabled-bg: @mantle;\n    }\n\n    & when not (@flat) {\n      --color-btn-@{item}-text: @base;\n      --color-btn-@{item}-bg: @color;\n      --color-btn-@{item}-icon: @base;\n\n      --color-btn-@{item}-hover-text: @base;\n      --color-btn-@{item}-hover-bg: lighten(@color, 10%);\n      --color-btn-@{item}-hover-border: lighten(@color, 10%);\n\n      --color-btn-@{item}-selected-text: @base;\n      --color-btn-@{item}-selected-bg: lighten(@color, 10%);\n      --color-btn-@{item}-selected-border: lighten(@color, 10%);\n\n      --color-btn-@{item}-disabled-text: @crust;\n      --color-btn-@{item}-disabled-bg: fadeout(@color, 60%);\n    }\n\n    --color-btn-@{item}-hover-icon: @base;\n\n    /* set these to transparent for the looks */\n    --color-btn-@{item}-border: transparent;\n    --color-btn-@{item}-shadow: 0 0 transparent;\n    --color-btn-@{item}-inset-shadow: 0 0 transparent;\n    --color-btn-@{item}-selected-shadow: 0 0 transparent;\n    --color-btn-@{item}-disabled-border: transparent;\n  }\n\n  #catppuccin(@lookup, @accent) {\n    @rosewater: @catppuccin[@@lookup][@rosewater];\n    @flamingo: @catppuccin[@@lookup][@flamingo];\n    @pink: @catppuccin[@@lookup][@pink];\n    @mauve: @catppuccin[@@lookup][@mauve];\n    @red: @catppuccin[@@lookup][@red];\n    @maroon: @catppuccin[@@lookup][@maroon];\n    @peach: @catppuccin[@@lookup][@peach];\n    @yellow: @catppuccin[@@lookup][@yellow];\n    @green: @catppuccin[@@lookup][@green];\n    @teal: @catppuccin[@@lookup][@teal];\n    @sky: @catppuccin[@@lookup][@sky];\n    @sapphire: @catppuccin[@@lookup][@sapphire];\n    @blue: @catppuccin[@@lookup][@blue];\n    @lavender: @catppuccin[@@lookup][@lavender];\n    @text: @catppuccin[@@lookup][@text];\n    @subtext1: @catppuccin[@@lookup][@subtext1];\n    @subtext0: @catppuccin[@@lookup][@subtext0];\n    @overlay2: @catppuccin[@@lookup][@overlay2];\n    @overlay1: @catppuccin[@@lookup][@overlay1];\n    @overlay0: @catppuccin[@@lookup][@overlay0];\n    @surface2: @catppuccin[@@lookup][@surface2];\n    @surface1: @catppuccin[@@lookup][@surface1];\n    @surface0: @catppuccin[@@lookup][@surface0];\n    @base: @catppuccin[@@look
```

Comment:
Really not a fan of this, contributing to this repo is already bad enough...

Reply:
I'm impartial to the addition of this, but how does this make contributing worse?

Reply:
Clone size

Reply:
Perhaps this would be better if it was handled as a release, on a weekly schedule, rather then how it is now.